diff --git a/public/javascripts/game/canvasPainter.js b/public/javascripts/game/canvasPainter.js
index bba67d6..9bea292 100644
--- a/public/javascripts/game/canvasPainter.js
+++ b/public/javascripts/game/canvasPainter.js
@@ -16,10 +16,11 @@ define(['parallax', 'backgroundRessources', '../../sharedConstants'], function (
 
   // Birds sprites
   var BIRDS_SPRITES = [
-    'images/clumsy.png',
-    'images/clumsy-blue.png',
-    'images/clumsy-red.png',
-    'images/clumsy-multi.png'
+    'images/coop-avatar.png',
+    // 'images/clumsy.png',
+    // 'images/clumsy-blue.png',
+    // 'images/clumsy-red.png',
+    // 'images/clumsy-multi.png'
   ];
 
   var that = {},
diff --git a/sharedConstants.js b/sharedConstants.js
index 9415bdc..592b1c3 100644
--- a/sharedConstants.js
+++ b/sharedConstants.js
@@ -3,7 +3,7 @@ var constant = {
 
   SERVER_PORT:              4242,
   SOCKET_PORT:              1337,
-  SOCKET_ADDR:              'http://localhost',
+  SOCKET_ADDR:              'http://192.168.1.237',
 
   SCREEN_WIDTH:             900,
   SCREEN_HEIGHT:            768,
